.eslintrc.js

```diff
@@ -1,14 +1,25 @@
 module.exports = {
   env: {
     es6: true,
-    node: true,
-    jquery: true,
+    node: true
   },
   extends: [
     "eslint:recommended",
     "plugin:node/recommended",
   ],
   overrides: [
+    {
+      files: [
+        'public/js/**/*.js'
+      ],
+      env: {
+        browser: true,
```

Comment:
Everything in public/js runs in a browser context, so moving config here so we don't need `/* eslint-env browser */` in any .js files.